keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/SubmittedDeliveryConfig.kt

```diff
@@ -33,10 +29,3 @@ data class SubmittedEnvironment(
   @Description("Optional locations that are propagated to any [resources] where they are not specified.")
   val locations: SubnetAwareLocations? = null
 )
-
-val DeliveryConfig.resources: Set<Resource<*>>
-  get() = environments.flatMapTo(mutableSetOf()) { it.resources }
-
-fun DeliveryConfig.matchingArtifact(reference: String, type: ArtifactType): DeliveryArtifact =
-  artifacts.find { it.reference == reference && it.type == type }
-    ?: throw NoMatchingArtifactException(name, type, reference)
```

Comment:
Moved to `DeliveryConfig` as they didn't depend on anything in this class (except `NoMatchingArtifcatException`, which I've worked around) and are useful elsewhere.